src/contentful/utils.js

```diff
@@ -39,7 +39,7 @@ export const getContentfulRequest = (type, contentQuery) => {
 export const formatImage = (image, assets) => {
   const ref = assets[image.sys.id];
   if (!ref) return null;
-  return `https:${ref.fields.file.url}`;
+  return `https:${ref.fields.file?.url}`;
```

Comment:
would rather return `null` instead of `https://` in error case

Reply:
Would you go for this instead? 
```
// ... 
if (!ref.fields.file?.url) return null;
return `https:${ref.fields.file.url}`;
// ...
```

Reply:
Comment: Yes
```js
const ref = assets[image.sys.id];
const url = ref?.fields?.file?.url;
if (!url) return null;

return `https:${url}`;
```